packages/shorebird_cli/lib/src/shorebird_flutter.dart

```diff
@@ -218,4 +218,31 @@ class ShorebirdFlutter {
     shorebirdEnv.flutterRevision = revision;
     useFlutterProgress.complete();
   }
+
+  /// Returns an human readable string from the received [flutterRevision] and
+  /// [flutterVersion].
```

Comment:
An example of the returned string would be helpful here.

Reply:
Added one! Lmk what you think